contents.json

```diff
@@ -5093,6 +5093,11 @@
       "category": "uitableview",
       "description": "Nested hierarchy of expandable/collapsible cells to easily build elegant discussion threads.",
       "homepage": "https://github.com/tsucres/SwiftyComments"
+    }, {
+      "title": "ESTabBarController",
+      "category": "tab",
+      "description": "ESTabBarController is a Swift model for customize UI, badge and adding animation to tabbar items. Support lottie!",
```

Comment:
@kingiol What do you think about replacing the description by **"A highly customizable TabBarController component, which is inherited from UITabBarController."**? Because we have to remove **"Swift"** and **"Support lottie!"** from the one.

Reply:
ok